WalletWasabi/JsonConverters/Bitcoin/FeeRateJsonConverterNg.cs

```diff
@@ -0,0 +1,27 @@
+using NBitcoin;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace WalletWasabi.JsonConverters.Bitcoin;
+
+public class FeeRateJsonConverterNg : JsonConverter<FeeRate>
+{
+	/// <inheritdoc />
+	public override FeeRate? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+	{
+		if (reader.TokenType != JsonTokenType.Number)
+		{
+			throw new JsonException("Expected a JSON number value.");
```

Comment:
Thanks, this is ok. Please document in PR description that `InvalidCastException` was changed to `JsonException` here and in other places it might have changed.